plugin/core/sessions.py

```diff
@@ -1786,6 +1790,14 @@ def _apply_text_edits(
             return
         apply_text_edits(view, edits, required_view_version=view_version)
 
+    def _check_select_sheets(self, sheets: List[sublime.Sheet]) -> None:
+        if len(sheets) > 1 == len(self.window.selected_sheets()):
+            self.window.select_sheets(sheets)
+
+    def _check_focus_sheet(self, sheet: Optional[sublime.Sheet]) -> None:
+        if sheet and sheet != self.window.active_sheet():
+            self.window.focus_sheet(sheet)
```

Comment:
Maybe name `_set_selected_sheets` and `_set_focused_sheet`. They do some checks too but is it important to convey? I think not.